File: maxOccurence/max-occurence.js
// Online Javascript Editor for free
// Write, Edit and Run your Javascript code using JS Online Compiler

var str = 'asdsfsfewrewrewuuuuuuuuuuuuuuuufdsfdfdgdfgerrte';

function checkStr ( str ){
    var arr = new Array( 26 ).fill(0);
    
    for(let i = 0 ; i < str.length; i++){
        arr[str[i].charCodeAt(0) - 'a'.charCodeAt(0)]++;
    }

    let maxx = arr[0];
    let val ;
    for(let i = 0 ; i < arr.length; i++){
        if( maxx < arr[i] ) {
            maxx = arr[i];
            
            val = String.fromCharCode(i + 'a'.charCodeAt(0));
        }
        // console.log(arr[i]);
    };

    console.log("Maxx Character : ",  val  ,maxx);
}

checkStr( str );

File: maxOccurence/maxOccurence.cpp
// Online C++ compiler to run C++ program online
#include <iostream>
using namespace std;

int main() {
    // Write C++ code here
    string str = "avcderttghnnnfgrtnnnnfrtrtnn";
    int n = 26;
    int arr[n] = {0};
    for(int i = 0; i < str.length(); i++){
        arr[str[i] - 'a']++;
    }
    int maxx = arr[0];
    int val;
    char ch;
    
    for(int i = 0; i < n; i++){
        if( maxx < arr[i]){
            maxx = arr[i];
            val = i + (int)'a';
            ch = char(val);
            
        } 
        cout << arr[i]<< " ";
    }
    cout << endl;
    
    cout << "MaxOccrenceCharacter : " << ch << " - "<< maxx << endl;
  
    
    

    return 0;
}